meta-genivi-dev/recipes-sota/aktualizr/files/aktualizr.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Aktualizr, SOTA client written in C++
+Wants=network-online.target
+After=network.target network-online.target rvi.service
+Requires=network-online.target
+
+[Service]
+RestartSec=5
+Restart=on-failure
+Environment=DBUS_SESSION_BUS_ADDRESS=unix:path=/run/dbus/system_bus_socket
```

Comment:
So sorry that I have to go back on what I just said about everything being perfect but why is the service running on session bus? Session is for services that run for each user session separately and AFAIK SOTA is not meant to be per-user service. Besides we are installing it as a system service so that makes it very inconsistent.